### PR TITLE
helloworld-en.ipynb: a simple assignment example.

### PR DESCRIPTION
**Pull request by salikh**

### CODE REVIEW

File: exercises/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+languages, denoted by the suffix of the notebook, e.g. "en" for English
+and "ja" for Japanese.
+
+Each assignment notebook should have a `course_info` entry in the notebook
+metadata section that identifies the specific assignment and the course that
+the assignment belongs to.
+
+    "metadata": {
+      "course_info": {
+	"course_name": "cs101",
```
Comment (yunabe) [inline exercises/README.md]: Inconsistent indents

File: exercises/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Programming exercises
+
+This directory contains the programming exercises
+and their autograder scripts together.
+
+## Installation of the client environment
+
+Install virtualenv. The command may differ depending on the system.
```
Comment (yunabe) [inline exercises/README.md]: To be clear, this is the requirement for students, right?

At this moment, it's okay.
But I think we need to check what are the typical environments and non-programming computer skills of our target students.

I feel `pip`, `pip3`, `virtualenv`, `./env/bin/activate` are overwhelming even for experienced non-Python programmer.

File: exercises/README.md (hunk 1 above)
Comment (salikh) [reply exercises/README.md]: Fixed.

File: exercises/README.md (hunk 2 above)
Comment (salikh) [reply exercises/README.md]: At this moment, this instruction is for people who want to develop on this project,
but you are right that we should also mind the requirements and setup for the student-side installation.

I will look up the current setup used by the UoT course and will try to replicate that as close as possible,
and update the README later.

**Pull request merged**